src/main/scala/cromwell/binding/expression/Evaluator.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding.expression
+
+import cromwell.parser.WdlParser.AstNode
+
+import scala.language.postfixOps
+import scala.util.Try
+
+trait Evaluator {
+  type T
```

Comment:
Scala seems to start with `A` for truly generic types like collections libraries or their HOFs.  Writing my first FSM from scratch with Tyburn I found it very helpful that Akka's `LoggingFSM` uses the mnemonic type variables `S` and `D` for State and Data; I didn't even need to click through in the IDE to see what these were.